### PR TITLE
Adding 3words processor (Geocoder) documentation

### PR DESCRIPTION
**Pull request by JeanODS**

### CODE REVIEW

File: source/sourcing_and_processing_data/processing.rst [hunk 1]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * Language
+    * Field containing the language in which the 3 word address. Default is English
```
Comment (juliendbg) [inline source/sourcing_and_processing_data/processing.rst]: the desired language for the 3 word address

File: source/sourcing_and_processing_data/processing.rst [hunk 2]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+    * Field
+    * No
+  * * Coordinates
+    * Field containing the geographical coordinates
```
Comment (juliendbg) [inline source/sourcing_and_processing_data/processing.rst]: coordinates to geocode

File: source/sourcing_and_processing_data/processing.rst [hunk 3]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+    * Field
+    * yes
+  * * Output field
+    * Field that will contain the produced 3 word address
```
Comment (juliendbg) [inline source/sourcing_and_processing_data/processing.rst]: -produced +resulting

File: source/sourcing_and_processing_data/processing.rst (hunk 1 above)
Comment (juliendbg) [reply source/sourcing_and_processing_data/processing.rst]: It's not a field, it's a string. Even if it's a FieldParam, it doesn't correspond to an actual field in the dataset.

File: source/sourcing_and_processing_data/processing.rst [hunk 4]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * Language
+    * Field containing the language in which the 3 word address. Default is English
+    * Field
```
Comment (juliendbg) [inline source/sourcing_and_processing_data/processing.rst]: String (or Text, see similar processors)

File: source/sourcing_and_processing_data/processing.rst [hunk 5]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * 3 word adress
+    * Field containing the 3 word address
+    * Field
+    * yes
+  * * Output field
+    * Field that will contain the produced coordinates
```
Comment (juliendbg) [inline source/sourcing_and_processing_data/processing.rst]: -produced +resulting

File: source/sourcing_and_processing_data/processing.rst [hunk 6]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+    * yes
+  * * Output field
+    * Field that will contain the resulting 3 word address
+    * Field
+    * yes
+
+Get coordinates from a 3 word address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert a 3 word address into coordinates. Requires a what3words API key to be configured by a domain administrator.
```
Comment (meganelacheny) [inline source/sourcing_and_processing_data/processing.rst]: "**This processor** convert**s** a 3 word address into **geographical** coordinates. **A what3words API key is required for this processor.**"

File: source/sourcing_and_processing_data/processing.rst [hunk 7]
```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
...
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * 3 word adress
+    * Field containing the 3 word address
+    * Field
+    * yes
+  * * Output field
+    * Field that will contain the resulting coordinates
```
Comment (meganelacheny) [inline source/sourcing_and_processing_data/processing.rst]: "resulting **geographical** coordinates"

**Pull request merged**